src/v3/src/components/PhoneAuthenticator/PhoneAuthenticator.tsx

```diff
@@ -194,7 +194,7 @@ const PhoneAuthenticator: UISchemaElementComponent<UISchemaElementComponentWithV
       >
         <Box
           width={showExtension ? 0.7 : 1}
-          marginRight={showExtension ? 2 : 0}
+          marginInlineEnd={showExtension ? 2 : 0}
```

Comment:
Is this lintable?

Reply:
Right now, stylelint does not seem to work with these types of properties, I looked into making this work in the other ticket but after research there seems to be blocked because; 
The syntax library for css in js has a bug preventing it from working altogether (there is an open ticket regarding the issue with no resolution). 

I looked into using eslint to prevent certain properties but it would require us to identify them all and I wasn't sure of our appetite to do this (we totally can if needed).